aml-development/ozp-center
# 20180508 update category bulk

AMLOS-533 & AMLOS-604
Bulk Category Update & User Interface Form

Description:

-Combines the Validation and Bulk Category Update features

-Displays information needed by the App Steward when determining the impact prior to deleting a category. Allows the App Steward to identify potential orphans, and displays the current status of category listing associations

-Allows multiple listing categories to be modified at one time


How to test code

Pull the branch into ozp-center

Open AML & log-in as bigbrother

Navigate to the Menu-->Manage --> Center Settings : Category Tab

Select a category and click the 'Bulk Update" button located on the grid toolbar.

The Bulk Update window is displayed

To verify the data: The displayed 'Listing' and 'number of category associations' can be obtained from the following Endpoint:
GET /api/category/7/listing/ (replace '7' with the appropriate category_id)

NOTE: The endpoint was tested in a previous PR.
See- Backend: PR#409 20180206 bulk category
[https://github.com/aml-development/ozp-backend/pull/409

To Update a category:
 Double click on category name and dropdown will appear.
 Select a category from the dropdown
 Press 'Save' button on toolbar


Please Review
@aml-development/ozp-front-team

Checklist

Read CONTRIBUTING.md
Write code to complete task
All tests must pass before merging the pull request
At least 2 people reviewed code
